### PR TITLE
enhance markers

### LINKED ISSUE #155: click on markers or bounding boxes
show name of layer

zoom to layer too

and decrease the zoom level that markers switch to bounding boxes. i.e. make boxes come on sooner

### PR DESCRIPTION
**Pull request by boney-bun**
markers show layer names and are clickable
note: this commit use leaflet 1.1.0

fix #155

### CODE REVIEW

File: templates/geosafe/analysis/dynamic_scripts/create/main.html [hunk 1]
```diff
@@ -235,13 +235,28 @@
...
+            });
+            marker.on('click', function (event) {
+                //map.removeLayer(hazard_exposure_marker_groups);
+                //map.addLayer(hazard_exposure_rect_groups);
+                /*
+                if(category == "hazard") {
+                    update_hazard_layer(layer_id, false);
+                } else if (category == "exposure") {
+                    update_exposure_layer(layer_id, false);
+                } */
```
Comment (lucernae) [inline templates/geosafe/analysis/dynamic_scripts/create/main.html]: Remove commented code, if not used

File: templates/geosafe/analysis/dynamic_scripts/create/main.html [hunk 2]
```diff
@@ -235,13 +235,28 @@
...
+                zoom_to_box(map, [data['layer_bbox_x0'], data['layer_bbox_y0'], data['layer_bbox_x1'], data['layer_bbox_y1']]);
+
+            });
+            hazard_exposure_marker_groups.addLayer(marker);
 
             //use later, particularly if zoom > 10
             bounds = [[data['layer_bbox_y0'], data['layer_bbox_x0']], [data['layer_bbox_y1'], data['layer_bbox_x1']]];
-            var rect_layer;
```
Comment (lucernae) [inline templates/geosafe/analysis/dynamic_scripts/create/main.html]: why it is deleted?
Is the same variable defined elsewhere globally?

**Review by lucernae — COMMENTED** [review]
Nice work @boney-bun 

LGTM with some comments.

**Comment by boney-bun** [comment]
code has just refactored.
i'll merge them.
thanks for your feedback @lucernae

**Pull request merged**